### ISSUE TITLE
prop-types warns on this.props.children

### ISSUE THREAD

**Issue by raybooysen** [yannickcr/eslint-plugin-react#180]
Should this warn on children when it's a built in prop from React?

Sample Component
```
export class Component extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div>
          {this.props.children}
      </div>
    );
  }
}
```

Warning:
```
warning  'children' is missing in props validation for Component  react/prop-types
```

**Comment by yannickcr** [comment]
Yes, there is a note [in the documentation](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md#as-for-exceptions) about the `children` case. You an use [the `ignore` option](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md#rule-options) if you don't want to document it in your component.

**Comment by raybooysen** [comment]
Thank you, I didn't spot this on the first read.
